MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -14,6 +14,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 

```

Comment:
This commit builds on top of some TPL changes made years ago for hyperv. It is a bugfix for those changes and probably needs to go into a REBASE&FF (or squashed) with those changes. The bugfix is for SCT compliance.

Reply:
It builds off of this: https://github.com/microsoft/mu_basecore/commit/c3acc1ff370b9f93c811a32025789611a7809c36

Reply:
I've updated the PR to include that commit.